mcux/devices/MIMXRT685S/MIMXRT685S_cm33.h

```diff
@@ -17417,7 +17417,7 @@ typedef struct {
 /*! @} */
 
 /* The count of INPUTMUX_PINT_SEL */
-#define INPUTMUX_PINT_SEL_COUNT                  (8U)
+#define INPUTMUX_PINTSEL_COUNT                  (8U)
```

Comment:
This change only seems to change the #define name, we could use the `INPUTMUX_PINT_SEL_COUNT` instead of changing to `INPUTMUX_PINTSEL_COUNT`

Reply:
@mmahadevan108 all the soc series uses **INPUTMUX_PINTSEL_COUNT** to get the count from header expect MIMXRT685S_cm33. this change is required to unify the macro among all the series.
![all_series_uses_INPUTMUX_PINTSEL_COUNT](https://user-images.githubusercontent.com/28673597/140502794-b1b89170-97b3-4db2-9b80-698cfd0e72af.png)